smac/tae/execute_ta_run_old.py

```diff
@@ -94,16 +84,18 @@ def run(self, config, instance=None,
                 quality = float(quality)
                 seed = int(seed)
 
-        if status in ["SAT", "UNSAT", "SUCCESS"]:
+        if status.upper() in ["SAT", "UNSAT", "SUCCESS"]:
             status = StatusType.SUCCESS
-        elif status in ["TIMEOUT"]:
+        elif status.upper() in ["TIMEOUT"]:
             status = StatusType.TIMEOUT
-        elif status in ["CRASHED"]:
+        elif status.upper() in ["CRASHED"]:
             status = StatusType.CRASHED
-        elif status in ["ABORT"]:
+        elif status.upper() in ["ABORT"]:
             status = StatusType.ABORT
-        elif status in ["MEMOUT"]:
+        elif status.upper() in ["MEMOUT"]:
             status = StatusType.MEMOUT
+        else:
+            status = StatusType.CRASHED
 
         if status in [StatusType.CRASHED, StatusType.ABORT]:
             self.logger.warn(
```

Comment:
Especially if we display the  warning like that.
As a user of SMAC I'd then expect that my algorithm crashed and would not start checking for typos or something like that.

SMAC should see the status as crashed but the user needs some more information about what went wrong. Maybe we could add a logging warning for unintelligible output without introducing a new StatusType?